wolfcrypt/src/wc_port.c

```diff
@@ -488,7 +488,7 @@ int wc_FileLoad(const char* fname, unsigned char** buf, size_t* bufLen,
     void* heap)
 {
     int ret;
-    size_t fileSz;
+    long int fileSz;
```

Comment:
'long int' -> 'long'

Reply:
Jacob hates bare `long` :grin:

Reply:
I don't like `long` or `long int`. Size can vary based on platform. This really should be `size_t`.